.github/workflows/releases.yml

```diff
@@ -26,6 +26,22 @@ jobs:
           args: release --release-notes=CHANGELOG.md
         env:
           GITHUB_TOKEN: ${{secrets.UPLOAD_GITHUB_TOKEN}}
+  cards:
+    needs: goreleaser
+    runs-on: ubuntu-latest
+    if: ${!contains(github.ref, "-")}
```

Comment:
`if` conditions don't need the `${{ }}` syntax; it's implicit for them.

You do, however, need to wrap the `if` expression in double quotes because otherwise the initial `!` character has special meaning in YAML.

Reply:
if I have to escape anyway, why not keep the `${}`?

Reply:
1. I'm not even sure if `if:` supports `${{ ... }}` in the first place, but you can try it! The [officially documented syntax](https://help.github.com/en/actions/reference/contexts-and-expression-syntax-for-github-actions) is `if: <expression>`, not `if: ${{ <expression> }}`
2. I think it should be `${{ }}` (double brackets) instead of `${ }`

Reply:
ah, i didn't realize the docs specifically said ${{}} was unneeded. switched to quotes.